src/raytracer/xmlreader.m

```diff
@@ -1,49 +1,52 @@
 function [CADOutput, materialSwitch] = xmlreader(filename, ...
     MaterialLibrary, referencePoint, r, IndoorSwitch)
-%XML redaer extracts the information of CAD file (AMF). The input of the
-%function is filename, the material database with all the material
-%parameters (Material_library), reference point (referencePoint) and distance limitation(r)
-%The output is extracted triangles (CADop),
-%and a boolean to know whether the material information is
-%present in the CAD file (switch1)
+%XML redaer extracts the information of CAD file (AMF). 
+% The input of the function is filename, the material database with all the 
+% material parameters (Material_library), reference point (referencePoint)
+% and distance limitation(r)
+% The output is extracted triangles (CADop),
+% and a boolean to know whether the material information is
+% present in the CAD file (switch1)
```

Comment:
improve documentation with the usual format

Reply:
done